apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -163,24 +170,30 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
           return onCancel(new DisabledTransactionBroadcastError());
         }
         onResult(operation);
-        // else not swap i.e card and sell we close the drawer
+      } else if (
+        (data.exchangeType === ExchangeType.SELL || data.exchangeType === ExchangeType.SELL_NG) &&
+        sourceCurrency
```

Comment:
Can we check we have sellId too. I do not know if any other old discovery app is using this flow.

Reply:
There is no such thing as `sellId`